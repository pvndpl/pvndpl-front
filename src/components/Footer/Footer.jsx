import style from "./Footer.module.css";
import Information from "./Information/Information";
import Navigation from "./Navigation/Navigation";

const Footer = () => {
    return (
        <footer className={style.footer}>
            <Information/>
            <Navigation/>
        </footer>
    );
}

export default Footer