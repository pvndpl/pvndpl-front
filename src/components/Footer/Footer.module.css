.footer {
    background: #ffffff;
    grid-area: footer;
    display: flex;
    justify-content: space-around;
}