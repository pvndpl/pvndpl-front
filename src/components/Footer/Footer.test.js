import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from './Footer';

describe('<Footer />', () => {
  test('it should mount', () => {
    const element = render(<Footer />);

    expect(element).toMatchSnapshot();

  });
});