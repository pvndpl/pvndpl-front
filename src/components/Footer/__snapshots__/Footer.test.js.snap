// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> it should mount 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <svg
      aria-hidden="true"
      id="__SVG_SPRITE_NODE__"
      style="position: absolute; width: 0; height: 0"
      xmlns="http://www.w3.org/2000/svg"
      xmlns:xlink="http://www.w3.org/1999/xlink"
    >
      <symbol
        fill="none"
        id="logo_vk_28"
        viewBox="0 0 28 28"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M4.546 4.546C3 6.093 3 8.582 3 13.56v.88c0 4.978 0 7.467 1.546 9.013C6.093 25 8.582 25 13.56 25h.88c4.978 0 7.467 0 9.013-1.547C25 21.907 25 19.418 25 14.44v-.88c0-4.978 0-7.467-1.547-9.014C21.907 3 19.418 3 14.44 3h-.88C8.582 3 6.093 3 4.546 4.546ZM6.7 9.7c.12 5.747 2.984 9.2 8.006 9.2h.285v-3.288c1.846.184 3.241 1.538 3.801 3.288H21.4c-.716-2.615-2.598-4.061-3.774-4.614 1.175-.681 2.828-2.339 3.223-4.586H18.48c-.514 1.823-2.038 3.481-3.489 3.638V9.7h-2.369v6.373c-1.469-.369-3.324-2.155-3.406-6.373H6.7Z"
          fill="currentColor"
          fill-rule="evenodd"
        />
      </symbol>
    </svg>
    <div>
      <footer
        class="footer"
      >
        <div
          class="brand"
        >
          <div
            class="infoBrand"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium logo css-ygf75t-MuiSvgIcon-root"
              data-testid="SportsBarIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 9h-1.56c.35-.59.56-1.27.56-2 0-2.21-1.79-4-4-4-.34 0-.66.05-.98.13-.82-.68-1.86-1.11-3.02-1.11-1.89 0-3.51 1.11-4.27 2.71C4.15 5.26 3 6.74 3 8.5c0 1.86 1.28 3.41 3 3.86V21h11v-2h2c1.1 0 2-.9 2-2v-6c0-1.1-.9-2-2-2zM7 10.5c-1.1 0-2-.9-2-2 0-.85.55-1.6 1.37-1.88l.8-.27.36-.76C8 4.62 8.94 4.02 10 4.02c.79 0 1.39.35 1.74.65l.78.65S13.16 5 13.99 5c1.1 0 2 .9 2 2h-3C9.67 7 9.15 10.5 7 10.5zM19 17h-2v-6h2v6z"
              />
            </svg>
            <div>
              <p
                class="infoBrandTitle"
              >
                ПИВАНДОПАЛА
              </p>
              <p
                class="infoBrandText"
              >
                Социальная сеть
              </p>
            </div>
          </div>
          <p
            class="infoText"
          >
            Социальная сеть «ПИВАНДОПАЛА» создана в 2022 году, наша цель — найти людей, любящих пиво!
          </p>
          <div
            class="infoLinkSocialNetwork"
          >
            <a
              class="linkSocialNetwork"
              href="https://www.youtube.com/channel/UClo75epUOSoGrOpoZkg1onw"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="YouTubeIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 15l5.19-3L10 9v6m11.56-7.83c.13.47.22 1.1.28 1.9.07.8.1 1.49.1 2.09L22 12c0 2.19-.16 3.8-.44 4.83-.25.9-.83 1.48-1.73 1.73-.47.13-1.33.22-2.65.28-1.3.07-2.49.1-3.59.1L12 19c-4.19 0-6.8-.16-7.83-.44-.9-.25-1.48-.83-1.73-1.73-.13-.47-.22-1.1-.28-1.9-.07-.8-.1-1.49-.1-2.09L2 12c0-2.19.16-3.8.44-4.83.25-.9.83-1.48 1.73-1.73.47-.13 1.33-.22 2.65-.28 1.3-.07 2.49-.1 3.59-.1L12 5c4.19 0 6.8.16 7.83.44.9.25 1.48.83 1.73 1.73z"
                />
              </svg>
            </a>
            <a
              class="linkSocialNetwork"
              href="https://github.com/pvndpl"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="GitHubIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                />
              </svg>
            </a>
            <a
              class="linkSocialNetwork"
              href="https://vk.com/belython"
            >
              <div
                class=" Icon Icon--28 Icon--w-28 Icon--h-28 Icon--logo_vk_28 "
                role="presentation"
                style="width: 28px; height: 28px;"
              >
                <svg
                  height="28"
                  style="display: block;"
                  viewBox="0 0 28 28"
                  width="28"
                >
                  <use
                    style="fill: currentColor;"
                    xlink:href="#logo_vk_28"
                  />
                </svg>
              </div>
            </a>
          </div>
        </div>
        <div>
          <p
            class="navigationText"
          >
            Запланировано
          </p>
          <div
            class="navigation"
          >
            <a
              class="navigationLink"
              href=""
            >
              Личная Страница
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Новости
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Страница Сообществ
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Страница Музыки
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Страница Фотографий
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Активность
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Друзья
            </a>
            <a
              class="navigationLink"
              href=""
            >
              Настройки
            </a>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      <div
        class="brand"
      >
        <div
          class="infoBrand"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium logo css-ygf75t-MuiSvgIcon-root"
            data-testid="SportsBarIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 9h-1.56c.35-.59.56-1.27.56-2 0-2.21-1.79-4-4-4-.34 0-.66.05-.98.13-.82-.68-1.86-1.11-3.02-1.11-1.89 0-3.51 1.11-4.27 2.71C4.15 5.26 3 6.74 3 8.5c0 1.86 1.28 3.41 3 3.86V21h11v-2h2c1.1 0 2-.9 2-2v-6c0-1.1-.9-2-2-2zM7 10.5c-1.1 0-2-.9-2-2 0-.85.55-1.6 1.37-1.88l.8-.27.36-.76C8 4.62 8.94 4.02 10 4.02c.79 0 1.39.35 1.74.65l.78.65S13.16 5 13.99 5c1.1 0 2 .9 2 2h-3C9.67 7 9.15 10.5 7 10.5zM19 17h-2v-6h2v6z"
            />
          </svg>
          <div>
            <p
              class="infoBrandTitle"
            >
              ПИВАНДОПАЛА
            </p>
            <p
              class="infoBrandText"
            >
              Социальная сеть
            </p>
          </div>
        </div>
        <p
          class="infoText"
        >
          Социальная сеть «ПИВАНДОПАЛА» создана в 2022 году, наша цель — найти людей, любящих пиво!
        </p>
        <div
          class="infoLinkSocialNetwork"
        >
          <a
            class="linkSocialNetwork"
            href="https://www.youtube.com/channel/UClo75epUOSoGrOpoZkg1onw"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="YouTubeIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 15l5.19-3L10 9v6m11.56-7.83c.13.47.22 1.1.28 1.9.07.8.1 1.49.1 2.09L22 12c0 2.19-.16 3.8-.44 4.83-.25.9-.83 1.48-1.73 1.73-.47.13-1.33.22-2.65.28-1.3.07-2.49.1-3.59.1L12 19c-4.19 0-6.8-.16-7.83-.44-.9-.25-1.48-.83-1.73-1.73-.13-.47-.22-1.1-.28-1.9-.07-.8-.1-1.49-.1-2.09L2 12c0-2.19.16-3.8.44-4.83.25-.9.83-1.48 1.73-1.73.47-.13 1.33-.22 2.65-.28 1.3-.07 2.49-.1 3.59-.1L12 5c4.19 0 6.8.16 7.83.44.9.25 1.48.83 1.73 1.73z"
              />
            </svg>
          </a>
          <a
            class="linkSocialNetwork"
            href="https://github.com/pvndpl"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="GitHubIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
              />
            </svg>
          </a>
          <a
            class="linkSocialNetwork"
            href="https://vk.com/belython"
          >
            <div
              class=" Icon Icon--28 Icon--w-28 Icon--h-28 Icon--logo_vk_28 "
              role="presentation"
              style="width: 28px; height: 28px;"
            >
              <svg
                height="28"
                style="display: block;"
                viewBox="0 0 28 28"
                width="28"
              >
                <use
                  style="fill: currentColor;"
                  xlink:href="#logo_vk_28"
                />
              </svg>
            </div>
          </a>
        </div>
      </div>
      <div>
        <p
          class="navigationText"
        >
          Запланировано
        </p>
        <div
          class="navigation"
        >
          <a
            class="navigationLink"
            href=""
          >
            Личная Страница
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Новости
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Страница Сообществ
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Страница Музыки
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Страница Фотографий
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Активность
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Друзья
          </a>
          <a
            class="navigationLink"
            href=""
          >
            Настройки
          </a>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
